beep/protocol/maccor_to_biologic_mb.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2019 Toyota Research Institute. All rights reserved.
+""" Parsing and conversion of maccor procedure files to arbin schedule files"""
+
+import os
+from beep.protocol import PROTOCOL_SCHEMA_DIR
+from monty.serialization import loadfn
+from collections import OrderedDict
+
+
+class MaccorToBiologicMb:
+    """
+    Collection of methods to convert maccor protocol files to biologic modulo bat protocol files
+    Differences in underlying hardware and software makes this impossible in some cases
+    or forces workarounds that may fail. Certain work-arounds may be attempted and are
+    documented under the conversion function.
+    """
+
+    def __init__(self):
+        BIOLOGIC_SCHEMA = loadfn(
+            os.path.join(PROTOCOL_SCHEMA_DIR, "biologic_mb_schema.yaml")
+        )
+        schema = OrderedDict(BIOLOGIC_SCHEMA)
+        self.blank_seq = schema["blank_seq"]
+
+    def __proc_step_to_seq(self, test_step, seq_num):
```

Comment:
I don't think the double leading underscore is necessary. The convention I have seen is just one underscore

Reply:
Ah, I didn't realize this! Can do!